### ISSUE TITLE
Different learned weights of models trained using slightly different strategies.

### ISSUE THREAD

**Issue by zwx8981** [pytorch/pytorch#49593]
I want to train two separate networks (without any parameters sharing) simultaneously (using the same data), say netA and netB. But it seems that the weights of the simultaneously trained netA are different from the individually trained netA.  All hyper-parameters are identical, and the random seed is also fixed. 

Anyone has some ideas about this issue? (Not important, but strange)

**Comment by VitalyFedyunin** [comment]
Hello! It is better to ask such kind of questions at https://discuss.pytorch.org/ I will close this issue, feel free to reopen it if you think that you need to report a bug or make feature proposal. Thanks.
